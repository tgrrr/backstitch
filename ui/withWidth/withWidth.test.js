import React from 'react'
import renderer from 'react-test-renderer'
import { withWidth } from '.'

it('withWidth: default', () => {
    const component = renderer.create(
        <withWidth>
            <>
                withWidth
            </>
        </withWidth>
    )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
