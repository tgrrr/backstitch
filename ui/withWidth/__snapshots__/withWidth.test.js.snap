// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withWidth: default 1`] = `
<withWidth>
  withWidth
</withWidth>
`;
